functions/src/algolia/index.ts

```diff
@@ -175,3 +126,36 @@ export const removeObjectFromIndices = async (objectId: string) => {
       }),
   ]);
 };
+
+/**
+ * Users who are visiting the app without logging in should be able to see posts on the map
+ * But they shouldn't be able to see the personal details of the posts.
+ * Information without personal details are stored in the unauthenticated requests index.
+ * This function generates a search key restricted to the unauthenticated requests index
+ *
+ * @returns {string} The secured key restricted to the unauthenticated requests index
+ */
+export const generateUnauthenticatedRequestsKey = () => {
```

Comment:
Can we add explicit return types for these two functions? It's unclear if searchKey is a single string or an object.

Reply:
okay, I will do that now